pkg/utils/watchJob.go

```diff
@@ -137,8 +136,8 @@ func (engineDetails EngineDetails) DeleteJobAccordingToJobCleanUpPolicy(experime
 			PropagationPolicy: &deletePolicy,
 		})
 		if deleteJob != nil {
-			return errors.Wrapf(err, "Unable to delete ChaosExperiment Job Name: %v, in namespace: %v, due to error: %v", experiment.JobName, experiment.Namespace, err)
+			return "", errors.Wrapf(err, "Unable to delete ChaosExperiment Job Name: %v, in namespace: %v, due to error: %v", experiment.JobName, experiment.Namespace, err)
```

Comment:
Can you return jobCleanup policy instead of null string , as you are using this value in event recorder?

Reply:
if it is unable to delete the job, then the event should not be added for that.